src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/high-risk-data-breaches/[type]/HighRiskBreachLayout.test.tsx

```diff
@@ -6,41 +6,80 @@ import { it, expect } from "@jest/globals";
 import { getAllByRole, render, screen } from "@testing-library/react";
 import { composeStory } from "@storybook/react";
 import { axe } from "jest-axe";
+import { setupJestCanvasMock } from "jest-canvas-mock";
 
 import Meta, {
   BankAccountStory,
   CreditCardStory,
   PinStory,
   SsnStory,
+  HighRiskBreachDonePasswordsNextStory,
+  HighRiskBreachDoneSecurityQuestionsNextStory,
+  HighRiskBreachDoneSecurityTipsNextStory,
+  HighRiskBreachDoneNoNextStepStory,
 } from "./HighRiskDataBreach.stories";
 
+beforeEach(() => {
+  setupJestCanvasMock();
+});
+
 it("passes the axe accessibility test suite for credit card breaches", async () => {
```

Comment:
I admire the goal of running accessibility tests on everything. However, the full axe suite takes a while to run, and so having a lot of these makes running our tests pretty slow (thereby disincentivising actually running them). Since the basic HTML structure for these different cases is, if I understand correctly, basically the same, it might be a sensible trade-off to only run one or two of them?

Reply:
I’ve removed the duplicate accessibility tests in [3d645c3](https://github.com/mozilla/blurts-server/pull/3753/commits/3d645c3da38446b66732f9aa04750c523dccb96d).